src/components/containers/Modal.vue

```diff
@@ -30,3 +32,11 @@ export default {
 	}
 };
 </script>
+
+<style lang="scss" scoped>
+@media screen and (max-width: 420px){
+	.mobile-model-content {
+		height: 300px;
```

Comment:
I think `max-height: 75%` should be more flexible

Reply:
yeah, but % is not working in this case.